namespaces/mson-namespace.md

```diff
@@ -564,7 +568,7 @@ Description is here! Properties to follow.
 ["object", {}, {}, [
   ["member", {}, {}, {
     "key": ["string", {}, {}, "p"],
-    "value": ["string", {}, {"typeAttributes": ["sample"]}, 42]
+    "value": ["string", {}, {"samples": [42]}, null]
```

Comment:
ok, but in this case you have to change it too in attributes specification (line no 103)